src/controllers/customer.js

```diff
@@ -48,6 +48,25 @@ const createCustomer = async (req, res) => {
     }
 }
 
+const updateCustomer = async (req,res) => {
+    const { id } = req.params;
+    const { name } = req.body;
+
+    const updatedCustomer = await prisma.customer.update({
+        where: {
+            id: Number(id),
+        },
+        data: {
+            name: "Jane",
```

Comment:
Wouldn't this always update a customer to have the name Jane?

Reply:
It would have to be name: name or just name would be sufficient on line 60